src/components/cc-header-addon/cc-header-addon.js

```diff
@@ -182,6 +184,12 @@ export class CcHeaderAddon extends LitElement {
           font-weight: bold;
         }
 
+        .messages {
+          align-items: center;
```

Comment:
**suggestion:** move `align-items: center` to the `cc-zone` selector. It doesn't do anything here because `.messages` is not flex / grid.
Moving it to the `cc-zone` will align the zone text with the country logo as shown below

in preview:
![image](https://github.com/user-attachments/assets/823a0d14-5932-485c-afa6-cf09f73fd944)

with the fix:
![image](https://github.com/user-attachments/assets/f2e290de-66c8-4807-b24e-4048e1e506f8)